deploy_pulp3/pulp3_db/files/pg_hba.conf

```diff
@@ -0,0 +1,14 @@
+# PostgreSQL Client Authentication Configuration File
+# ===================================================
+#
+# Refer to the "Client Authentication" section in the PostgreSQL
+# documentation for a complete description of this file.
+
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+
+# "local" is for Unix domain socket connections only
+local   all             all                                     trust
```

Comment:
I would recommend `local pulp {{ pulp_user }} peer`  as the permissions for this and the host connections.